Python/class/API.py

```diff
@@ -0,0 +1,148 @@
+import requests,json,time,datetime
+
+
+class Isense:
+
+    def __init__(self,projectID,contributorKey,contributorName):
+        self.projectID = projectID
+        self.contributorKey = contributorKey
+        self.contributorName = contributorName
+        self.baseUrl = 'http://rsense-dev.cs.uml.edu/api/v1/';
```

Comment:
baseUrl is never modified - each instance of Isense API doesn't need it's own.  Granted, there will probably ever only be one instance, but it's more of a best practices kind of thing.